pubspec.yaml

```diff
@@ -108,7 +108,7 @@ dependencies:
   linagora_design_flutter:
     git:
       url: git@github.com:linagora/linagora-design-flutter.git
-      ref: master
+      ref: apply-google-fonts
```

Comment:
do we need PR for that ?

Reply:
Pls review my PR: 
- https://github.com/linagora/linagora-design-flutter/pull/30

After that I'll back it to ref master